internal/stream/metrics.go

```diff
@@ -50,4 +51,10 @@ func init() {
 	workerComponentRestartCount = promauto.NewCounterVec(prometheus.CounterOpts{
 		Name: "metro_worker_component_restart_count",
 	}, []string{"env", "component", "topic", "subscription"})
+
+	workerPushEndpointLatency = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "metro_worker_push_endpoint_latency_seconds",
+		Help:    "Time taken to get response from push endpoint",
+		Buckets: prometheus.ExponentialBuckets(0.001, 1.25, 300),
```

Comment:
could not understand the reasoning behind these buckets? why spaced at 1.25 and 1.75?

Reply:
Bucket spacing is used same as push endpoint metric.